README.rst

```diff
@@ -1,6 +1,9 @@
 .. image:: https://travis-ci.com/amitdev/lru-dict.svg?branch=master
     :target: https://travis-ci.com/amitdev/lru-dict
 
+.. image:: https://github.com/amitdev/lru-dict/actions/workflows/tests.yml/badge.svg
+    :target: https://github.com/amitdev/lru-dict/actions/workflows/tests.yml
```

Comment:
This will be made available once it's merged.
See that commit from my fork to see how it looks
85ff84ec7f9682bfcad50f05f1abb5484731f1c5
https://github.com/AndreMiras/lru-dict/actions/workflows/tests.yml/badge.svg